cli/src/commands/commit.rs

```diff
@@ -67,6 +67,9 @@ pub(crate) struct CommitArgs {
         add = ArgValueCompleter::new(complete::modified_files),
     )]
     paths: Vec<String>,
+    // TODO: Delete in jj 0.40.0+
+    /// (deprecated, use `jj metaedit --update-author`)
+    ///
```

Comment:
nit: since we add `hide = true`, I think we can just leave the doc string unmodified. I'm okay with either way.